src/patternfly/demos/PageTest/examples/page-test-example1.hbs

```diff
@@ -0,0 +1,117 @@
+{{#> page-test}}
+
+{{#> background-image}}
+{{/background-image}}
+
+{{#> page}}
+
+{{#> page-section pf-l-page__main-section--modifiers="pf-m-dark"}}
```

Comment:
I'd use --modifier even if there are multiples just to keep it consistent.(4 lines down same thing)